doc/gui/examples/controls/metric-simple.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+# -----------------------------------------------------------------------------------------
+# To execute this script, make sure that the taipy-gui package is installed in your
+# Python environment and run:
+#     python <script>
+# -----------------------------------------------------------------------------------------
+from taipy.gui import Gui
+
+delta_value = 20
+min_value = 50
+max_value = 150
+threshold = 100
+
+page = """
+<|90|metric|show_value=true|min={min_value}|max={max_value}|delta={delta_value}|threshold={threshold}|format=%d km/h|delta_format=%dkm/h|width=800px|height=500px|>
```

Comment:
No need for Show_value= true (default)

Reply:
Using formats in the simple case is too much. Please drop formats as well.
That is taken care of in the `-format` example.

Reply:
resolved